packages/fxa-auth-server/lib/routes/password.js

```diff
@@ -94,7 +119,9 @@ module.exports = function (
           }
 
           if (password.clientVersion === 2) {
-            const unwrappedKb = await password.unwrap(emailRecord.wrapWrapKbVersion2);
+            const unwrappedKb = await password.unwrap(
```

Comment:
Did our prettier line length config change?

Reply:
Not that I know.  My setup is fairly vanilla so any auto-formatting by VS Code or pre-commit scripts I'd just trust and add.

Reply:
Doesn't look like this was auto-formatted lately, we also missing this file for async/await conversion it seems.